.travis.yml

```diff
@@ -1,8 +1,16 @@
-language: python
+services:
+  - docker
 
+language: python
 python:
     - '3.5'
 
+deploy:
+  provider: script
+  script: ./docker/push_image.sh
+  on:
+    tags: true
+
```

Comment:
Just as a personal style, I like putting the `deploy` key below the `jobs` key, since the `deploy` stuff only runs after all the other steps pass.